.github/workflows/release.yml

```diff
@@ -173,10 +175,8 @@ jobs:
     needs: [
         build-android, build-ios,
         build-linux-x64,
-        # build-macos-x64,
-        build-macos-aarch64,
-        # build-windows-x64, build-windows-aarch64
-    ]
+        build-macos-x64,
+        build-macos-aarch64] # build-windows-x64, build-windows-aarch64
```

Comment:
can you keep both mac builds on the same line and windows in the new line?

Reply:
done